bin/summariser.py

```diff
@@ -171,6 +171,7 @@ def runprocess(db_config_file, config_file):
             log.warning("The summariser may not start again until it is removed.")
 
         log.info(LOG_BREAK)
+        db.db.close()
```

Comment:
The connection should be closed before we do last bit of program cleanup, so I think just after the `finally` would be better.